dips/dip-8.md

```diff
@@ -15,14 +15,14 @@ An extension of the Off-Chain protocol to provide support for more advanced merc
 # Abstract / Motivation
 ---
 
-Version 0 of the Off-Chain Protocol is described in [DIP 1](https://dip.diem.com/dip-1/).  Version 1 as described here is an extension of the Off-Chain Protocol and adds features to support more advanced functionality - particularly targeted to support merchant use-cases.  This is inclusive of pull payments, recurring payments, and auth/capture flows.
+Versions 1/2 of the Off-Chain Protocol are described in [DIP 1](https://dip.diem.com/dip-1/).  Version 3 as described here is an extension of the Off-Chain Protocol and adds features to support more advanced functionality - particularly targeted to support merchant use-cases.  This is inclusive of pull payments, recurring payments, and auth/capture flows.
 
 ---
 # Specification
 ---
 
 # Fund Pull Pre-Approval
-### *VERSION SUPPORT: Supported only in version 1 of off-chain APIs*
+### *VERSION SUPPORT: Supported only in version 3 of off-chain APIs*
```

Comment:
is this version 3 match the version number in API path? version 2 uses v2, this is v3?

Reply:
Yeah, that's what I was thinking

Reply:
I was confused by this because at the moment we don't have v3 of that. Does it make more sense to say "This is not yet supported in off chain APIs but will soon be" ?

Reply:
There is no way to understand what version a VASP is supporting, so it's all or none, if some VASPs implemented v3, how they may communicate with others? some time use v2, some time use v3 ?

Reply:
Let me give an example, for vasp a, and vasp b:
1. vasp a calls vasp b for fund pre-approval, url: <base_url>/v3/command
2. vasp a calls vasp b for payment object referenced to the fund pre-approval, url: <base_url>/v3/command
3. now if there is payment from vasp a to vasp b without fund pre-approval, what url it should call? <base_url>/v3? or v2?